src/routes/+layout.svelte

```diff
@@ -22,6 +22,33 @@
 	});
 </script>
 
+<svelte:head>
+	<!-- HTML Meta Tags -->
+	<title>Swiss Cloud Native Day 2023</title>
+	<meta name="description" content="" />
```

Comment:
Here, I would put the same as we have for the `og:description` tag.

Reply:
Oops, missed that. Will fix it (and probably factor out the text into a variable :-)

Reply:
Done